src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -16,7 +16,11 @@
 #include <string>
 #include <string.h>
 #include <sys/stat.h>
-#ifdef HAVE_SYS_SYSCTL_H
```

Comment:
On a second note, for some reason the `HAVE_SYS_SYSCTL_H` cmake-checks fails, but in the process I discovered we also have a redundant `HAVE_SYSCTL` check which does *not* fail. And it used for the very same purpose: guarding `#include <sys/sysctl.h>` statements.

I don't know why we have 2 different ways to check for this same thing, but right now, if I were to have a choice, I would vote on eliminating the `HAVE_SYS_SYSCTL_H` check, as this should work on FreeBSD (and hopefully NetBSD too).

Sounds OK?